product_assortment/models/ir_filters.py

```diff
@@ -23,6 +23,12 @@ def _get_default_is_assortment(self):
     model_id = fields.Selection(
         default=lambda x: x._get_default_model())
 
+    partner_ids = fields.Many2many(
+        comodel_name='res.partner',
+        required=True,
```

Comment:
Why required?? You are adding a new behavior but can not do this new fields as required...